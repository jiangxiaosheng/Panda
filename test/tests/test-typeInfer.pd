func main() {
	var a = 1
	var b = 2.0
	var c =  "hello"
	var d = false
}