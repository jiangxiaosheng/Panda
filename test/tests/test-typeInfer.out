Sementically checked program: 


void main() {
var a: int(int : 1)
var b: float(float : 2.)
var c: string(string : hello)
var d: bool(bool : false)
}

